include/debug.h

```diff
@@ -1 +1,6 @@
+#ifndef __MEGA65_DEBUG_H
+#define __MEGA65_DEBUG_H
+
 void debug_msg(char* m);
+
+#endif
```

Comment:
Sometimes people do
```
#endif /* __MEGA65_DEBUG_H */
```
To make it easier to see what is closed, which can be useful in cases when the header file is long.
Matter of taste, `__MEGA65_DEBUG_H__` can be seen as more symmetric, but it is a matter of taste.

Reply:
Good idea, fixed (by github co-pilot ;-)). Most existing headers didn't add trailing __ so I went the path of least changes.